inc/Engine/Optimization/RUCSS/Database/Tables/UsedCSS.php

```diff
@@ -69,9 +69,11 @@ protected function set_schema() {
 	/**
 	 * Delete all used_css which were not accessed in the last month.
 	 *
+	 * @param int $delete_delay number of months before deleting.
+	 *
 	 * @return bool|int
 	 */
-	public function delete_old_used_css() {
+	public function delete_old_used_css( $delete_delay = 1 ) {
```

Comment:
Use the filter value here, and make sure it's a positive int and not zero value before using it in the query